### ISSUE TITLE
New translation: Finnish

### ISSUE THREAD

**Issue by akx** [squidfunk/mkdocs-material#769]
``` jinja
{% macro t(key) %}{{ {
  "language": "fi",
  "clipboard.copy": "Kopioi leikepöydälle",
  "clipboard.copied": "Kopioitu leikepöydälle",
  "edit.link.title": "Muokkaa tätä sivua",
  "footer.previous": "Edellinen",
  "footer.next": "Seuraava",
  "meta.comments": "Kommentit",
  "meta.source": "Lähdekodi",
  "search.language": "fi",
  "search.placeholder": "Hae",
  "search.result.placeholder": "Kirjoita aloittaaksesi haun",
  "search.result.none": "Ei täsmääviä dokumentteja",
  "search.result.one": "1 täsmäävä dokumentti",
  "search.result.other": "# täsmäävää dokumenttia",
  "search.tokenizer": "[\s\-]+",
  "skip.link.title": "Hyppää sisältöön",
  "source.link.title": "Mene repositoryyn",
  "toc.title": "Sisällysluettelo"
}[key] }}{% endmacro %}
```

The translation for "repository" is as-is, but it's a technical term without a very good non-silly Finnish term.

**Comment by squidfunk** [comment]
Thanks, awesome! But `repositoryyn` is correct and not a typo? Frankly, for me Finnish mostly looks like someone smacked on the keyboard ;-) Words with four `Ä` - just crazy. Anyway, was hoping that somebody would supply a translation becaus we have Finnish search support but no translations, so many many thanks for doing that!

**Comment by akx** [comment]
Yes, it's correct grammar-wise, it's just using the English word for "repository". :)

**Comment by squidfunk** [comment]
Ok great, will integrate it and make a release the next days!

**Comment by squidfunk** [comment]
Merged and released as part of 2.7.3.
